src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/PseHelper.java

```diff
@@ -0,0 +1,149 @@
+                    r.getValue().getNumber() + " " + r.getValue().getSendNotificationTitle()
+                )
+            )
+            .collect(Collectors.toList()));
+    }
+
+    /**
+     * Gets the selected SendNotificationTypeItem.
+     * @param caseData contains all the case data
+     * @return the select application in GenericTseApplicationTypeItem
+     */
+    public static SendNotificationTypeItem getSelectedSendNotificationTypeItem(CaseData caseData) {
+        String selectedAppId = caseData.getPseRespondentSelectOrderOrRequest().getSelectedCode();
+        return caseData.getSendNotificationCollection().stream()
+            .filter(s -> s.getValue().getNumber().equals(selectedAppId))
+            .findFirst()
+            .orElse(null);
+    }
+
+    /**
+     * Format Markup for displaying Hearing, case management order or request.
+     * @param sendNotificationType Target SendNotification Subject
+     * @return Hearing, case management order or request Markup
+     */
+    public static String formatOrdReqDetails(SendNotificationType sendNotificationType) {
+        return String.format(
+            ORDER_APP_MARKUP,
+            sendNotificationType.getSendNotificationTitle(),
+            sendNotificationType.getSendNotificationSelectHearing() == null
+                ? ""
+                : sendNotificationType.getSendNotificationSelectHearing().getSelectedLabel(),
+            sendNotificationType.getDate(),
+            defaultString(sendNotificationType.getSendNotificationCaseManagement()),
+            defaultString(sendNotificationType.getSendNotificationResponseTribunal()),
+            defaultString(sendNotificationType.getSendNotificationSelectParties()),
+            defaultString(sendNotificationType.getSendNotificationAdditionalInfo()),
+            sendNotificationType.getSendNotificationUploadDocument() == null
+                ? ""
+                : sendNotificationType.getSendNotificationUploadDocument().stream()
+                    .map(d -> String.format(
+                        ORDER_APP_DOC_MARKUP,
+                        d.getValue().getShortDescription(),
+                        Helper.getDocumentMatcher(d.getValue().getUploadedDocument().getDocumentBinaryUrl())
+                            .replaceFirst(""),
+                        d.getValue().getUploadedDocument().getDocumentFilename()
+                    ))
+                    .collect(Collectors.joining()),
+            defaultString(sendNotificationType.getSendNotificationCaseManagement()),
+            CASE_MANAGEMENT_ORDER.equals(sendNotificationType.getSendNotificationCaseManagement())
+                ? defaultString(sendNotificationType.getSendNotificationWhoCaseOrder())
+                : defaultString(sendNotificationType.getSendNotificationRequestMadeBy()),
+            defaultString(sendNotificationType.getSendNotificationFullName()),
+            sendNotificationType.getSendNotificationNotify()
+        );
+    }
```

Comment:
This method is fairly complex, can you refactor to make this more readable? I like the use of `defaultString`, good spot